.github/workflows/main.yml/pics.yml

```
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
    branches: [ main ]
```suggestion
{Link: @continuous integration]




```

```

---

> ```suggestion
>     branches: [ main ]
> ```suggestion
> {Link: @continuous integration]
> 
> 
> 
> 
> ```
> 
> ```
>